styx-cli/src/main/java/com/spotify/styx/cli/PrettyCliOutput.java

```diff
@@ -113,13 +126,15 @@ public void printBackfill(Backfill backfill) {
 
   private void printBackfillHeader() {
     System.out.println(String.format(BACKFILL_FORMAT,
-                                     "BACKFILL ID",
-                                     "HALTED",
-                                     "ALL TRIGGERED",
+                                     "BACKFILL_ID",
```

Comment:
These changes don't seem to be very necessary since this is pretty print anyway.